engine/src/main/java/org/terasology/network/events/PingFromClientEvent.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.network.events;
+
+import org.terasology.entitySystem.event.Event;
+import org.terasology.network.ServerEvent;
+
+/**
+ * This event, from client to server, implement one part of ping to all clients
```

Comment:
Which part? :-) This is probably actually good enough but it is written a little awkwardly. Maybe:

"This event handles the ping from a client to the server"